modules/claims_api/app/workers/claims_api/vbms_uploader.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+
+module ClaimsApi
+  class VbmsUploader
+    include Sidekiq::Worker
+
+    def perform(power_of_attorney_id)
+      power_of_attorney = ClaimsApi::PowerOfAttorney.find(power_of_attorney_id)
+      uploader = ClaimsApi::PowerOfAttorneyUploader.new(power_of_attorney_id)
+      uploader.retrieve_from_store!(power_of_attorney.file_data[:filename])
+      filepath = "#{uploader.file.file}/#{power_of_attorney.file_data['filename']}"
+
+      upload_token_response = fetch_upload_token(
+        filepath: filepath,
+        file_number: power_of_attorney.auth_headers['va_eauth_pnid']
```

Comment:
I know if the benefits intake api it can be file number or ssn, is it always ssn in this case or should we support a filenumber input?

Reply:
Its a good question. The vbms gem asks for a file number, but I am not sure if under the hood they allow either.

Reply:
Hrm who can we talk to to figure this out, seems like it would be relevant